runtime/integration-tests/src/authority.rs

```diff
@@ -266,78 +266,19 @@ fn test_authority_module() {
 				false,
 				Box::new(call.clone())
 			));
-			System::assert_last_event(Event::Authority(orml_authority::Event::Scheduled(
-				OriginCaller::system(RawOrigin::Root),
-				6,
-			)));
+			System::assert_last_event(Event::Authority(orml_authority::Event::Scheduled {
+				origin: OriginCaller::system(RawOrigin::Root),
+				index: 6,
+			}));
 
 			assert_ok!(Authority::cancel_scheduled_dispatch(
 				Origin::root(),
 				Box::new(frame_system::RawOrigin::Root.into()),
 				6
 			));
-			System::assert_last_event(Event::Authority(orml_authority::Event::Cancelled(
-				OriginCaller::system(RawOrigin::Root),
-				6,
-			)));
+			System::assert_last_event(Event::Authority(orml_authority::Event::Cancelled {
+				origin: OriginCaller::system(RawOrigin::Root),
+				index: 6,
+			}));
 		});
 }
-
-#[test]
```

Comment:
Why remove this?

Reply:
this must be a mistake (perhaps merge error).
I'll add this test back and double check agian